### DISCUSSION TITLE
YAML module in pure Lua

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/textadept#310] [Show and tell]
I really needed it, so I did it again ;-)

https://github.com/paaguti/textadept-yaml

**Comment by eric1234** [comment]
Forgive me if I'm just dense but how is this pure Lua. It seems instead of calling to a yaml library compiled from C it is calling out to a yaml CLI program written in python. It seems neither would be pure Lua...

**Comment by paaguti** [comment]
By pure Lua I mean that when installing the module you don't need to cmake
anything.
Yes, you need external programs; so fine if we don't want to call this pure
Lua. But there are luarocks out there which work on the same principle.


On Sat, 17 Dec 2022 at 18:33, eric1234 ***@***.***> wrote:

> Forgive me if I'm just dense but how is this pure Lua. It seems instead of
> calling to a yaml library compiled from C it is calling out to a yaml CLI
> program written in python. It seems neither would be pure Lua...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orbitalquark/textadept/discussions/310#discussioncomment-4438552>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4CGK4PD3DABIOI63DLWNX2NFANCNFSM6AAAAAATA6CIVU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fragen sind nicht da um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

Headaches with a Juju log:
unit-basic-16: 09:17:36 WARNING juju.worker.uniter.operation we should run
a leader-deposed hook here, but we can't yet

**Reply by ghost** [reply]
Yeah, I understand what you said. You seem to be not English speaker, so am I, so we are easier to understand each other. Instead of having to implement it in C++ code, you are calling the external programs via Lua code without having to write a single C++ code :smile:
